From 257ac9bedd19bddbebc4683afdf6c193469062ea Mon Sep 17 00:00:00 2001
From: Maximilian Marx <mmarx@wh2.tu-dresden.de>
Date: Sat, 23 May 2026 16:26:50 +0200
Subject: [PATCH 1/2] Require name and institution in preference polls

---
 AKPreference/models.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/AKPreference/models.py b/AKPreference/models.py
index cf4ccba..31255d9 100644
--- a/AKPreference/models.py
+++ b/AKPreference/models.py
@@ -13,9 +13,9 @@ class EventParticipant(models.Model):
         verbose_name_plural = _('Participants')
         ordering = ['name']
 
-    name = models.CharField(max_length=64, blank=True, verbose_name=_('Nickname'),
+    name = models.CharField(max_length=64, blank=False, required=True, verbose_name=_('Nickname'),
                             help_text=_('Name to identify a participant by (in case of questions from the organizers)'))
-    institution = models.CharField(max_length=128, blank=True, verbose_name=_('Institution'), help_text=_('Uni etc.'))
+    institution = models.CharField(max_length=128, blank=False, required=True, verbose_name=_('Institution'), help_text=_('Uni etc.'))
 
     event = models.ForeignKey(to=Event, on_delete=models.CASCADE, verbose_name=_('Event'),
                               help_text=_('Associated event'))
-- 
2.51.2

